sass/mixins/status/status.scss

```diff
@@ -0,0 +1,32 @@
+@import "palette.scss";
+
+@mixin status($type) {
+  display: inline-block;
+  padding: 7px 10px;
+  border-radius: 25px;
+  text-align: center;
+
+  &.alt {
```

Comment:
alt as in alternative is style related in my eyes. The states you describe above active, disabled etc should be variants of the component. Although this is open to discussion.